app/services/seed_database.rb

```diff
@@ -34,16 +32,16 @@ def dual_seed(model, hash_array)
         email: hash[:email],
         password: "test1234",
       )
-      model.create!(
+      model.create(
         hash.merge({ user_id: user.id })
             .except(:email),
-      )
+      ) # TODO: need to bang ! this when postcodes valid, enable rubocop Rails/SaveBang
```

Comment:
Why can't we have the bang here?

Reply:
good spot, that has been updated with the Save/Bang disable, have replaced